sklearn/neural_network/tests/test_mlp.py

```diff
@@ -59,7 +60,7 @@ def test_alpha():
     absolute_sum = lambda x: np.sum(np.abs(x))
 
     for alpha in alpha_values:
-        mlp = MLPClassifier(hidden_layer_sizes=10, alpha=alpha, random_state=1)
+        mlp = MLPClassifier(hidden_layer_sizes=10, alpha=alpha, random_state=1, max_iter=500)
```

Comment:
You are right @GaelVaroquaux: the test is about 2x longer. Will switch to `ignore_warngins` then.